components/eventCard/EventItem.tsx

```diff
@@ -0,0 +1,292 @@
+import React, { useState } from "react"
+import {
+  Alert,
+  Image,
+  StyleSheet,
+  Text,
+  TouchableWithoutFeedback,
+  View
+} from "react-native"
+import { Event } from "@lib/events/Event"
+import { Divider } from "react-native-elements"
+import { Shadow } from "react-native-shadow-2"
+import tinycolor from "tinycolor2"
+import {
+  Menu,
+  MenuOptions,
+  MenuOption,
+  MenuTrigger
+} from "react-native-popup-menu"
+import { placemarkToFormattedAddress } from "@lib/location"
+import { TouchableOpacity } from "react-native-gesture-handler"
+import { compactFormatMiles } from "@lib/DistanceFormatting"
+import { MaterialCommunityIcon, MaterialIcon } from "../common/Icons"
+
+interface Props {
+  event: Event
+}
+
+const EventItem = ({ event }: Props) => {
+  const [isMenuOpen, setMenuOpen] = useState(false)
+  const numAttendees = 1 // Eventually will come from event object
+  const distance = 0.5 // Eventually will be calculated from the location given in the event object
+  const lightEventColor = tinycolor(event.color).lighten(25).toString()
+  let dismiss: () => void
+
+  const alertDelete = () => {
+    dismissMenu()
+    Alert.alert("Delete this event?", undefined, [
+      { text: "Delete", style: "destructive", onPress: dismiss },
+      { text: "Cancel", style: "cancel", onPress: dismiss }
+    ])
+  }
+
+  // Will probably need to use a modal instead to have user select/type in reason for report
+  const alertReport = () => {
+    dismissMenu()
+    Alert.alert("Report this event?", undefined, [
+      { text: "Report", style: "destructive", onPress: dismiss },
+      { text: "Cancel", style: "cancel" }
+    ])
+  }
```

Comment:
Is it possible that we can get the menu stuff into a reusable component? We'll need it for the detail view.